### PR TITLE
fix(components): [select] add popper-options prop

### PR DESCRIPTION
**Pull request by ryuhangyeong**
Please make sure these boxes are checked before submitting your PR, thank you!

https://github.com/element-plus/element-plus/discussions/11631

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/select.md [hunk 1]
```diff
@@ -117,45 +117,47 @@ If the binding value of Select is an object, make sure to assign `value-key` as
...
+| loading-text          | displayed text while loading data from server                                                                    | string                                     | —                   | Loading          |
+| no-match-text         | displayed text when no data matches the filtering query, you can also use slot `empty`                           | string                                     | —                   | No matching data |
+| no-data-text          | displayed text when there is no options, you can also use slot `empty`                                           | string                                     | —                   | No data          |
+| popper-class          | custom class name for Select's dropdown                                                                          | string                                     | —                   | —                |
+| popper-options        | Customized popper option see more at [popper.js](https://popper.js.org/docs/v2/)                                 | object                                     | —                   | —                |
```
Comment (ryuhangyeong) [inline docs/en-US/component/select.md:146]: ## Self Review

add `popper-options` props description

**Review by btea — APPROVED** [review]
LGTM

File: docs/en-US/component/select.md (hunk 1 above)
Comment (chenxch) [reply docs/en-US/component/select.md:146]: Try to execute a format to see if the indentation will be the same as the original

**Review by chenxch — APPROVED** [review]

File: docs/en-US/component/select.md (hunk 1 above)
Comment (ryuhangyeong) [reply docs/en-US/component/select.md:146]: image

I ran `npm run format` . There are no existing changes.

Comment (chenxch) [reply docs/en-US/component/select.md:146]: ok.

**Pull request merged**